sphinx/screens-detail/create-tribe/create-tribe/src/main/java/chat/sphinx/create_tribe/ui/CreateTribeFragment.kt

```diff
@@ -163,7 +172,7 @@ internal class CreateTribeFragment: SideEffectFragment<
                     if (it.isNullOrEmpty()) {
                         null
                     } else {
-                        it.toString().toLong() * Companion.MILLISECONDS_IN_AN_HOUR
+                        it.toString().toLong() * MILLISECONDS_IN_AN_HOUR
```

Comment:
Try/Catch blocks needed for use of `.toLong()` as it will throw a `NumberFormatException`. Alternatively you can utilize `.toLongOrNull()` and handle it that way.